freud/diffraction.py

```diff
@@ -13,7 +13,6 @@
 """
 
 import logging
-from importlib.util import find_spec
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.